src/kirby/components/segmented-control/segmented-control.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, EventEmitter, Output, Input } from '@angular/core';
+
+import { SegmentItem } from './segment-item';
+
+@Component({
+  selector: 'kirby-segmented-control',
+  templateUrl: './segmented-control.component.html',
+  styleUrls: ['./segmented-control.component.scss'],
+})
+export class SegmentedControlComponent {
+  @Output() segmentClicked: EventEmitter<any> = new EventEmitter();
```

Comment:
Use `SegmentItem` as selected item:
```suggestion
  @Output() segmentClicked: EventEmitter<SegmentItem> = new EventEmitter();
```